tests/lib/util/npm-util.js

```diff
@@ -20,71 +20,63 @@ const assert = require("chai").assert,
 // Tests
 //------------------------------------------------------------------------------
 
-describe("npmUtil", function() {
+describe("npmUtil", () => {
 
     let sandbox;
 
-    beforeEach(function() {
+    beforeEach(() => {
         sandbox = sinon.sandbox.create();
     });
 
-    afterEach(function() {
+    afterEach(() => {
         sandbox.verifyAndRestore();
     });
 
-    describe("checkDevDeps()", function() {
+    describe("checkDevDeps()", () => {
         let installStatus;
 
-        before(function() {
+        before(() => {
             installStatus = npmUtil.checkDevDeps(["debug", "mocha", "notarealpackage", "jshint"]);
         });
 
-        it("should not find a direct dependency of the project", function() {
+        it("should not find a direct dependency of the project", () => {
             assert.isFalse(installStatus.debug);
         });
 
-        it("should find a dev dependency of the project", function() {
+        it("should find a dev dependency of the project", () => {
             assert.isTrue(installStatus.mocha);
         });
 
-        it("should not find non-dependencies", function() {
+        it("should not find non-dependencies", () => {
             assert.isFalse(installStatus.notarealpackage);
         });
 
-        it("should not find nested dependencies", function() {
+        it("should not find nested dependencies", () => {
             assert.isFalse(installStatus.jshint);
         });
 
-        it("should return false for a single, non-existent package", function() {
+        it("should return false for a single, non-existent package", () => {
             installStatus = npmUtil.checkDevDeps(["notarealpackage"]);
             assert.isFalse(installStatus.notarealpackage);
         });
 
-        it("should handle missing devDependencies key", function() {
-            sandbox.stub(shell, "test", function() {
-                return true;
-            });
-            sandbox.stub(fs, "readFileSync", function() {
-                return JSON.stringify({
-                    private: true,
-                    dependencies: {}
-                });
-            });
+        it("should handle missing devDependencies key", () => {
+            sandbox.stub(shell, "test", () => true);
```

Comment:
This one's not your fault, just that this isn't idiomatic `sinon` usage. Should be:

```js
sandbox.stub(shell, "test").returns(true);
```

Up to you if you want to fix this one. Same would apply to the next line as well.